x-pack/plugins/index_management/server/plugin.ts

```diff
@@ -77,6 +77,19 @@ export class IndexMgmtServerPlugin implements Plugin<IndexManagementPluginSetup,
       }
     );
 
+    features.registerElasticsearchFeature({
+      id: PLUGIN.id,
+      management: {
+        data: ['index_management'],
+      },
+      privileges: [
+        {
+          requiredClusterPrivileges: ['monitor'],
```

Comment:
I think we should also add the `manage_templates` privilege here, as index management also encompasses managing index and component templates.